server/src/controller_test.py

```diff
@@ -26,6 +29,24 @@ def json(self):
     data = json.loads(controller.get_slack_users().body)
     assert data == {'message': 'Slack API error', 'ok': False}
 
+def test_get_app_users():
+    mockUser = mocks.MockUser("testuser", "testpassword", "tester", 0, "sudoers")
+    mockRequest = {"username": "testuser", "team": "sudoers"}
+
+    result = controller.get_app_users(mockRequest, mocks.MockFilterProvider(mockUser))
+    data = json.loads(result.body)
+    assert data['ok'] == True
```

Comment:
actually, there's really no need to compare any of these values to booleans or None. you can just do 'assert data['ok']' or 'assert not data['ok']'